### PR TITLE
fix(nx): skip rollup overrides

### PR DESCRIPTION
**Pull request by sapphi-red**
This makes nx pass: https://github.com/vitejs/vite-ecosystem-ci/actions/runs/6728289738/job/18287407127

### CODE REVIEW

File: utils.ts [hunk 1]
```diff
@@ -272,9 +272,11 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
...
 		] ||= `${options.vitePath}/packages/plugin-legacy`
 
-		const vitePackageInfo = await getVitePackageInfo(options.vitePath)
-		if (vitePackageInfo.dependencies.rollup?.version && !overrides.rollup) {
-			overrides.rollup = vitePackageInfo.dependencies.rollup.version
+		if (overrides.rollup !== false) {
+			const vitePackageInfo = await getVitePackageInfo(options.vitePath)
+			if (vitePackageInfo.dependencies.rollup?.version && !overrides.rollup) {
+				overrides.rollup = vitePackageInfo.dependencies.rollup.version
+			}
```
Comment (sapphi-red) [inline utils.ts:279]:
```suggestion
		const vitePackageInfo = await getVitePackageInfo(options.vitePath)
		//skip if `overrides.rollup` is `false`
		if (vitePackageInfo.dependencies.rollup?.version && overrides.rollup == null) {
			overrides.rollup = vitePackageInfo.dependencies.rollup.version
		}
```

File: utils.ts [hunk 2]
```diff
@@ -273,9 +273,11 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 		] ||= `${options.vitePath}/packages/plugin-legacy`
 
 		const vitePackageInfo = await getVitePackageInfo(options.vitePath)
-		if (vitePackageInfo.dependencies.rollup?.version && !overrides.rollup) {
+		//skip if `overrides.rollup` is `false`
+		if (vitePackageInfo.dependencies.rollup?.version && overrides.rollup == null) {
 			overrides.rollup = vitePackageInfo.dependencies.rollup.version
 		}
+		}
```
Comment (sapphi-red) [inline utils.ts:280]:
```suggestion
```

File: utils.ts [hunk 3]
```diff
@@ -273,7 +273,11 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 		] ||= `${options.vitePath}/packages/plugin-legacy`
 
 		const vitePackageInfo = await getVitePackageInfo(options.vitePath)
-		if (vitePackageInfo.dependencies.rollup?.version && !overrides.rollup) {
+		//skip if `overrides.rollup` is `false`
```
Comment (sapphi-red) [inline utils.ts:276]:
```suggestion
		// skip if `overrides.rollup` is `false`
```

File: utils.ts [hunk 4]
```diff
@@ -273,7 +273,11 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 		] ||= `${options.vitePath}/packages/plugin-legacy`
 
 		const vitePackageInfo = await getVitePackageInfo(options.vitePath)
-		if (vitePackageInfo.dependencies.rollup?.version && !overrides.rollup) {
+		// skip if `overrides.rollup` is `false`
+		if (
+			vitePackageInfo.dependencies.rollup?.version &&
+			overrides.rollup == null
```
Comment (dominikg) [inline utils.ts:279]:
```suggestion
			overrides.rollup !== false
```
to avoid overrides.rollup=true   resulting in rollup not being overridden

**Review by dominikg — APPROVED** [review]

**Pull request merged**